go.mod

```diff
@@ -33,5 +33,6 @@ require (
 replace (
 	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
 	// Workaround for https://github.com/golang/go/issues/30831 and fallout.
-	github.com/golang/lint => github.com/golang/lint v0.0.0-20190227174305-8f45f776aaf1
+	github.com/golang/lint => github.com/golang/lint v0.0.0-20181217174547-8f45f776aaf1
+	willnorris.com/go/imageproxy v0.8.1-0.20190326225038-d4246a08fdec => willnorris.com/go/imageproxy v0.8.1-0.20190422234945-d4246a08fdec
```

Comment:
is this a result of updating go dependencies? Don't think `imageproxy` is being used anywhere 🤔

Reply:
This is a result of using Go 1.13. Because 1.13 validates the commit hash with the timestamp, we need these replace blocks to correctly get the dependency. `imageproxy` is a transitive dependency that is used somewhere else in the module graph.

I have kept this in a separate commit so that they remain separate.